### DISCUSSION TITLE
feature: Set Shop/Bank Scroll Bar Limit

### DISCUSSION THREAD

**Discussion by NikoCummings** [AscensionGameDev/Intersect-Engine#1859] [Ideas]
### Description

It would be nice if we could set a limit to the scroll bar in shops and banks via a config file or something. I'm referring to the horizontal scroll bar that shows up when you have a lot of money in your inventory and try to purchase a stackable item. I think currently it just sets itself based upon how many of the stackable item you can afford based on your cash. This can get ugly if a player has 1m in their inventory and try to purchase an item that's cheap (like 10g). So lets say the scrollbar allows you to purchase up to 99999 of an item. Imagine trying to use that scrollbar left and right. If you move it just the tiniest hair, you have jumped up 1,000 in quantity.

### Intended Use-Case

A better user experience for players and an easy to use scroll bar.

### Duplicate Check

- [X] This feature request is not a duplicate to the best of my knowledge.

**Comment by Arufonsu** [comment]
Currently, https://github.com/AscensionGameDev/Intersect-Engine/pull/1720 address the bank deposit scroll sliders behavior, and i'll soon address the withdrawals from the bank so the scroll sliders have limited values in relation to the exact max number of items that the inventory is capable to hold !

**Comment by Arufonsu** [comment]
should be solved by #1992
